src/Sentry.Unity/Extensions/SentryOptionsExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿using UnityEngine;
+using CompressionLevel = System.IO.Compression.CompressionLevel;
+
+namespace Sentry.Unity.Extensions
+{
+    internal static class SentryOptionsExtensions
+    {
+        public static void ConfigureLogger(this SentryOptions sentryOptions, UnitySentryOptions unitySentryOptions)
+        {
+            if (unitySentryOptions.Logger == null)
+            {
+                return;
+            }
+
+            sentryOptions.Debug = unitySentryOptions.Debug;
+            sentryOptions.DiagnosticLogger = unitySentryOptions.Logger;
+            sentryOptions.DiagnosticLevel = unitySentryOptions.DiagnosticsLevel;
+        }
+
+        public static void ConfigureRelease(this SentryOptions sentryOptions, UnitySentryOptions unitySentryOptions)
+            // Uses the game `version` as Release
+            => sentryOptions.Release = unitySentryOptions.Release is { } release
+                ? release
+                : Application.version;
+
+        public static void ConfigureEnvironment(this SentryOptions sentryOptions, UnitySentryOptions unitySentryOptions)
+            => sentryOptions.Environment = unitySentryOptions.Environment is { } environment
+                ? environment
+                : Application.isEditor
+                    ? "editor"
+                    // TODO: should we remove #if?
+#if DEVELOPMENT_BUILD

```

Comment:
We'll need this to get through via a script that compiles with Unity to use their compiler flags.

Reply:
Still not sure what you want to do with this. Your first comment suggests to leave it "as it", the second one is "to remove".

Overall, what value do you expect for `sentryOptions.Environment` field in **the current** PR?

Reply:
With this, if it wasn't clear:
> We'll need this to get through via a script that compiles with Unity to use their compiler flags.

I meant that `DEVELOPMENT_BUILD` doesn't work here. Unless I'm missing something.
Is this defined by Unity?